app/lib/package/api_export/exported_api.dart

```diff
@@ -98,40 +98,37 @@ final class ExportedApi {
   Future<void> _gcPrefix(String prefix, Set<String> allPackageNames) async {
```

Comment:
Change here simplified a bit, just list all files, extract package name from the object name and if the package doesn't exist, we delete it if it's older than `_minGarbageAge`